pkg/flow/flow.go

```diff
@@ -137,6 +138,10 @@ type Flow struct {
 // New creates and starts a new Flow controller. Call Close to stop
 // the controller.
 func New(o Options) *Flow {
+	return newController(newModuleRegistry(), o)
+}
+
+func newController(modReg *moduleRegistry, o Options) *Flow {
```

Comment:
`newController` feels like an overloaded term here.  Also `New` might need to be renamed to `NewRoot`  (that feels like a terrible name) or `NewController`. It feels odd to know when to use which New/newController.

Reply:
It might make more sense to rename it in the future when we combine packages but I don't think we should change public API names because of internal implementation details. 

I can document when to call newController internally.